docs/Manpage.md

```diff
@@ -899,6 +899,17 @@ present, "revision 1" will be added.
 * `--message`:
   Append *`message`* to the default commit message.
 
+### `bump-unversioned-casks` [*`options`*] [*`tap`*]
+
+Check all casks with unversioned URLs in a given *`tap`* for updates.
+
+* `-n`, `--dry-run`:
+  List what would be done, but do not actually do anything.
```

Comment:
What does it do? It looks like it should just "check" but it's unclear what happens as part of that check?